src/mono/wasm/debugger/BrowserDebugProxy/EvaluateExpression.cs

```diff
@@ -231,6 +231,13 @@ private string ConvertJSToCSharpLocalVariableAssignment(string idName, JToken va
                         typeRet = "string";
                         break;
                     }
+                    case "symbol":
+                     {
+                         var str = value?.Value<string>()?.Replace("\'", "")?.LastOrDefault();
```

Comment:
This doesn't look right.

1. we use `symbol` for other types too, like delegates. (grep for `TSymbol` in the tests)
2. extracting the char value this way is brittle

Looking at the object here, it has a `value` that gets set here:

https://github.com/dotnet/runtime/blob/30f30a3e5b6be09eacf7ea4cf054576753ed99ca/src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs#L1766-L1770

It is using `description` as the value too, like `"97 'a'"` (first arg for `CreateJObject<>`).

Instead, we can use the char as the value in `CreateJObjectForChar`, like:

```csharp
            char charValue = Convert.ToChar(value);
            var description = $"{value} '{charValue}'";
            return CreateJObject<char>(charValue, "symbol", description, true);
```

And here, you will have the character available as the value.

Do confirm that the visible values in chrome, and VS are still correct.

Reply:
It is correct, I left TSymbol only for delegates and made a separate one, dedicated for chars only.

Reply:
We can't use `type=char` for the json object. The allowed types are dictated by https://chromedevtools.github.io/devtools-protocol/tot/Runtime/#type-RemoteObject

So, on the proxy side you would have to do like I suggested. And on the tests side, you can have a `TChar` but one that checks for `symbol`.

The `T*` methods on the tests side are just to make it easier to match the json objects.